elliott/scripts/jira_issue_tool.py

```
@@ -0,0 +1,331 @@
+    # Dump all JIRA field name-to-ID mappings (useful for migration)
+    ./jira_issue_tool.py dump-fields
+"""
+
+import argparse
+import json
+import os
+import sys
+from typing import Optional
+
+from jira import JIRA
+import ipdb
```

Ruff (0.15.2)

[warning] Import for `ipdb` found

(T100)